### PR TITLE
Fall back to hardcoded token_salt in dev/test environment

### PR DESCRIPTION
**Pull request by denisw**
This makes it easier to get started.

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -14,6 +14,11 @@ config :jumubase, JumubaseWeb.Endpoint,
   watchers: [node: ["node_modules/brunch/bin/brunch", "watch", "--stdin",
                     cd: Path.expand("../assets", __DIR__)]]
 
+# Configure Phauxth authentication
+config :phauxth,
+  token_salt: System.get_env("PHAUXTH_TOKEN_SALT") || "1J9hSjU4",
```
Comment (richeterre) [inline config/dev.exs]: Let's default to something like `String.duplicate("x", 8)` to make it perfectly clear that no real secrets are getting hard-coded here 😅

File: config/dev.exs [hunk 2]
```diff
@@ -14,6 +14,11 @@ config :jumubase, JumubaseWeb.Endpoint,
...
+  endpoint: JumubaseWeb.Endpoint
```
Comment (richeterre) [inline config/dev.exs]: Should we set the endpoint once in `config.exs`? Now that line is in three different files.

File: config/dev.exs (hunk 1 above)
Comment (denisw) [reply config/dev.exs]: Good idea. Although` "xxxxxxxx"` is shorter and equally clear. 😉

File: config/dev.exs (hunk 2 above)
Comment (denisw) [reply config/dev.exs]: Good idea! Will change that.

File: config/dev.exs (hunk 1 above)
Comment (richeterre) [reply config/dev.exs]: Well, in some cases the token length matters (and might be a lot longer) and with that style you immediately see the length. But for 8 chars I agree.

**Pull request merged**